### ISSUE TITLE
New Install of OpenGTS, and Calamp DCS Listening Port not showing up in netstat -plnt.

### ISSUE THREAD

**Issue by scmachines** [tananaev/traccar#1975]
I'm not sure if I'm posting in the correct place, but I have a related issue. I'm trying to get some Calamp GPS devices to talk to my new install of OpenGTS on Ubuntu 14.04. I had no Calamp-specific files to work with, so I'm running the server as "template.jar" with the Calamp port and other settings pasted into the dcservers.xml file. When I start the DCS server, I get a pid and bin/psjava shows the server is running - but sudo netstat -plnt does not show the listening port. I have also tried starting servers like aspicore (for test purposes) that a jar file exists for - but no listening port shows up. These are the instructions I've cobbled together in my implementation:http://linoxide.com/security/setup-opengts-gps-tracking-ubuntu-centos/ and http://www.geotelematic.com/docs/DCServer_CalAmp.pdf . We are just in a testing phase with two GPS devices inside of vehicles and configured to send their data to port 20500. The vendor is not able to be much help with any software, and I am not a software developer by profession. Is there something I'm missing that is preventing the port from listening? Thanks...

**Comment by tananaev** [comment]
It is incorrect place. This repository is for Traccar project. It has nothing to do with OpenGTS.

There is a way you can use Traccar server to receive data from your CalAmp devices and write them into OpenGTS database.

**Comment by scmachines** [comment]
Thank you, Tananaev, for your response. If a Traccar server would work for this purpose, then I'd be interested to learn more. I might begin by trying to install Traccar on the Ubuntu server.
What I'm trying to accomplish with my colleagues in the Czech Republic is to listen for, and receive, GPS data from the Calamp devices and get the data onto the server. I'm going to read up on Traccar and begin installing it on the server.

**Comment by tananaev** [comment]
Installation should be very easy. Just download the package and check readme file for instructions.

If you want to integrate Traccar with OpenGTS, see following page:

https://www.traccar.org/opengts/

**Comment by scmachines** [comment]
I was able to install traccar just as you said, and data seems to be reaching the server: 
pkts bytes target         prot opt in      out     source                  destination         
64  9322 ACCEPT     udp  --  any    any     anywhere             anywhere             udp dpt:5082
This is great progress when compared to where I was at before!

I've also modified the traccar.xml (within the conf directory) exactly as described in the link above - deleting the database queries and copy/pasting the sql ones into the file via nano, then saving the changes. I've also restarted the server a couple of times since then to observe the results. 

I noticed that the mysql database is not filling up with any data. I've checked that the root username and password are in there. Am I missing a component to get the calamp data into the database? Thanks...
